File: Bubble_Sort/bubbleSort.php
<?php
function bubbleSort(array $arr): array
{
    $n = count($arr);

    for ($i = 0; $i < $n - 1; $i++) {
        $swapped = false;

        for ($j = 0; $j < $n - $i - 1; $j++) {
            if ($arr[$j] > $arr[$j + 1]) {
                $temp = $arr[$j];
                $arr[$j] = $arr[$j + 1];
                $arr[$j + 1] = $temp;

                $swapped = true;
            }
        }

        if (!$swapped)
            break;
    }

    return $arr;
}

$arr = [8, 10, 7, 1, 4, 6, 2, 5, 9, 3];
$sortedArr = bubbleSort($arr);

print_r($sortedArr);

File: Bubble_Sort/bubble_sort.py
def bubble_sort(arr: list):
    n = len(arr)

    for i in range(n):
        swapped = False

        for j in range (n - i - 1):
            if arr[j] > arr[j + 1]:
                arr[j], arr[j + 1] = arr[j + 1], arr[j]

                swapped = True

        if not swapped:
            break


arr = [8, 10, 7, 1, 4, 6, 2, 5, 9, 3]
bubble_sort(arr)

print(arr)
